React-Projects/job-portal/src/components/EditJob/EditJob.module.scss

```diff
@@ -1,65 +1,65 @@
-$post-job-margin-top: 150px;
-$post-job-color: #333;
-$post-job-border-color: #ccc;
-$post-job-primary-color: #007bff;
+$margin-top: 150px;
+$color: #333;
+$border-color: #ccc;
+$primary-color: #007bff;
 
```

Comment:
@codal-kcolaco Please create one file color.scss and add all the colors there and re-use it everywhere

Reply:
@abhi-codal  Understood